README.md

```diff
@@ -28,15 +26,15 @@ This site is a work in progress and the documentation currently resides in their
 
 > The smart contract system is a collection of solidity contracts deployed on the Trustlines Blockchain. Transfers within the Trustlines Network are executed by the smart contracts. All trustlines (i.e. credit lines and balances between users) are notarized on the Trustlines Blockchain. Furthermore, the smart contracts enforce the rules determining how trustlines can be created, used and updated.
 
-### Cloning the repository
+### Cloning the repository	
```

Comment:
No changes to this file should be made and the made changes add multiple unnecessary empty spaces.